cdk.json

```diff
@@ -43,34 +43,8 @@
       "STICKY":"TRUE",
       "DESIRED_TASK_COUNT":"2"
     },
-    "dev-no-sticky": {
```

Comment:
how did dev-no-sticky and  dev-sticky ever get deployed to an AWS account without a matching GH action deployment in [aws-deploy.yaml](https://github.com/Sage-Bionetworks/data_curator-infra/blob/dev/.github/workflows/aws-deploy.yml)

Reply:
ohh wait, i see it now.  it's all under `cdk-deploy-dev`.  You'll want to remove `NO-STICKY` and `STICKY` ones as well.

Reply:
thanks for catching that @zaro0508. I just removed them from `aws-deploy.yml`